.travis.yml

```diff
@@ -24,6 +24,7 @@ before_install:
   - phpenv config-add ci/travis.php.ini
 
 before_script:
+  - phpenv config-add ci/travis.php.ini
```

Comment:
why is this in before_install and before_script now? Should be needed only once.

Reply:
@dereuromark Could you please clarify, should this change be reverted? Or it needs to be re-checked somehow?

Reply:
reverted, this is already in line 24.

Reply:
why was this added a 2nd time?